content/references-rest-api.md

```diff
@@ -147,6 +147,38 @@ Example response:
 }
 ```
 
+## Users
+
+### GET users
+
+Users endpoint can be used to retrieve users' ids who are subscribed to a specific presence channel.
+
+```
+/apps/<id>/channels/<name>/users
+```
+
+#### Request
+
+Get users is a plain request so there isn't any additional parameters.
+The request is only valid to presence channels.
+
+#### Response
+
+A successful response contains a JSON object. The JSON contains a list of user ids. 
+It returns an empty list if the channel name is currently unoccupied.
```

Comment:
I think third should be changed to "[...] if the channel is currently unoccupied."